elan/rpc/rpc.go

```diff
@@ -46,6 +46,9 @@ const timeout = 2 * time.Minute
 
 var log = cli.MustGetLogger()
 
+// emptyHash is the sha256 hash of the empty file.
+const emptyHash = "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855"
```

Comment:
nit: could do `var emptyHash = digest.Empty.Hash`

Reply:
Done (although mildly sad it now isn't a constant)